utils/Lint/cpplint.py

```diff
@@ -62,6 +62,17 @@
 #    ctor(s?), dtor, friend declarations, methods, member vars)
 #
 
+# This file has been modified to more closely enforce
+#   HYPED's style-guide, which is itself adopted from
+#   Google's own c++ style guide.
+#
+# Changes tracked so far:
```

Comment:
Do we want to keep track of changes within the file itself? Why not just use the git log? Just worried about these comments in the file not being updated/forgotten about.

Reply:
I personally do not care. Is this a request for change? Or just a comment? Or a start of a long discussion that will prevent this PR from merging for weeks?

Reply:
I originally put that there for two reasons: 1. I didn't know how much I could track using git 2. I thought it'd be nice to state that there is a difference between the original Google's lint script and this one, and also what the differences are.

Now, I know that I can look at all changes made to a file and what each change did via git log and git diff, so 1. isn't really a reason anymore. That being said, what do you think about 2. ? I guess if someone really wants to know what changes we specifically made, then they can also use git diff and git log (assuming they have/can get git), so we don't have to track the changes there.  

I'm also worried about the list not being updated, so I'll remove it for now, but can bring it back later if it'd be useful.